### PR TITLE
CN-exec: Upgrade single file runtime test script

### PR DESCRIPTION
**Pull request by dc-mak**
Ownership checking is optionally enabled, and there's now an option for running quietly (only outputting the output directory on success), a help flag and more robust command checking.

### CODE REVIEW

**Comment by dc-mak** [comment]
This is compatible with https://github.com/rems-project/cn-tutorial/commit/115464f217f81ea032144d49e96c3a991cd56861

**Comment by cp526** [comment]
(Think this is for @rbanerjee20 to review.)

File: runtime/libcn/dune [hunk 1]
```diff
@@ -16,3 +16,9 @@
   (libcn.a as runtime/libcn.a))
  (section lib)
  (package cn))
+
+(install
+ (files
+  (libexec/cn-runtime-single-file.sh as runtime/libexec/cn-runtime-single-file.sh))
```
Comment (dc-mak) [inline runtime/libcn/dune:22]: https://dune.readthedocs.io/en/stable/reference/dune/install.html#including-files-in-the-install-stanza

**Comment by septract** [comment]
@rbanerjee20 should be reviewed and merged before we can merge #458

**Review by rbanerjee20 — APPROVED** [review]
Looks good to me.

**Comment by septract** [comment]
Thanks! Could you merge this? I don’t have credentials on the Cerberus repo. On Aug 7, 2024, at 7:09 AM, Rini Banerjee ***@***.***> wrote:﻿
@rbanerjee20 approved this pull request.

Looks good to me.

**Comment by rbanerjee20** [comment]
@septract done!

**Comment by septract** [comment]
Thanks! I’ll fix up the other PR to point to the correct branch. On Aug 7, 2024, at 7:48 AM, Rini Banerjee ***@***.***> wrote:﻿
@septract done!

**Pull request merged**